packages/capacitor-plugin/package.json

```
@@ -0,0 +1,89 @@
+{
+  "name": "@capacitor/filesystem",
+  "version": "7.1.0",
```

Minor: 7.1.0 will be the future plugin version, maybe it shouldn't be updated in this PR yet.

```suggestion
  "version": "7.0.0",
```

---

fixed